widget/embedded/src/hooks/useSwapInput.ts

```diff
@@ -112,15 +127,34 @@ export function useSwapInput(): UseSwapInput {
       }
       fetchQuote(requestBody)
         .then((res) => {
+          const sortQuotes = sortRoutesBy(sortStrategy, res.results);
+          const quote = sortQuotes.length
+            ? {
+                requestAmount: res.requestAmount,
+                ...sortQuotes[0],
+              }
+            : null;
           setLoading(false);
-          setQuote(res);
-          throwErrorIfResponseIsNotValid(res);
-          const quoteWarning = generateQuoteWarnings(res, {
-            fromToken,
-            toToken,
-            userSlippage,
-            tokens,
+          throwErrorIfResponseIsNotValid({
+            diagnosisMessages: res.diagnosisMessages,
+            requestId: quote?.requestId || '',
+            swaps: quote?.swaps,
           });
+          updateSomeQuoteState('quotes', res);
+          const sortedQuoteTags = getDefaultTag(
```

Comment:
I believe using the name 'selectedTag' would be more fitting.

Reply:
fixed